templates/platform_landing_zone/lib/architecture_definitions/alz_custom.alz_architecture_definition.yaml

```diff
@@ -0,0 +1,71 @@
+name: alz_custom
```

Comment:
When I originally setup the accelerator, there was a reason I allowed overriding the `alz` name for the architecture. I don't recall now if it was just convenience or there was another reason. This was definitely added on purpose, but the reason now escapes me: https://github.com/Azure/alz-terraform-accelerator/blob/3cc6c19f9598976688ca0c50046ad8b003be7208/templates/platform_landing_zone/terraform.tf#L25

Reply:
It could just be to support the fact we copy the files from the library here: https://azure.github.io/Azure-Landing-Zones/accelerator/startermodules/terraform-platform-landing-zone/options/management-groups/

Reply:
I'm wondering what the impact of renaming management group IDs is here? Would renaming a management group ID result in it being created twice as it is defined with a different ID in the `alz` library?

E.g. rename `- id: alz` to `- id: contoso`. Will alz still be created because it is defined in the `alz` library?

Reply:
The only MGs that are created are those defined in the architecture definition supplied to the provider data source.

The library will have two definitions, the default `alz` and the local `alz_custom`.

Custom is selected and therefore will be created. The other architecture is ignored.

Reply:
Ok, sounds good. I hadn't appreciated it worked like that.

Reply:
Let's talk this though as I am not sure I understand the scenario

Reply:
I removed it from the config file as people kept getting confused about what it was and thinking it was related the management group names.